### DISCUSSION TITLE
Is there going to be mesh shader support for OpenGL

### DISCUSSION THREAD

**Discussion by 4Jed** [GPUOpen-Drivers/AMD-Gfx-Drivers#20] [Q&A]
its not that big of a deal but in minecraft there is a mod that uses mesh shaders to increase the render distance and using the gpu for rendering them instead of the cpu/ ram rendering them 
so im hoping this is planned but im guessing not because i found a previous question similar to this 7 months ago

**Comment by Sotumney** [comment]
https://github.com/GPUOpen-Drivers/AMD-Gfx-Drivers/issues/4#issuecomment-2290797051

Soon!
